pkg/activator/net/throttler.go

```diff
@@ -333,16 +333,33 @@ func (rt *revisionThrottler) handleUpdate(throttler *Throttler, update revisionD
 		for newDest := range update.Dests {
 			tracker, ok := trackersMap[newDest]
 			if !ok {
-				tracker = &podIPTracker{dest: newDest}
+				if rt.containerConcurrency == 0 {
+					tracker = &podIPTracker{dest: newDest, b: newInfiniteBreaker(rt.logger)}
+					tracker.b.UpdateConcurrency(1) // Unlock the pod breaker.
+				} else {
+					tracker = &podIPTracker{
+						dest: newDest,
+						b: queue.NewBreaker(queue.BreakerParams{
+							QueueDepth:      throttler.breakerParams.QueueDepth,
+							MaxConcurrency:  int(rt.containerConcurrency),
+							InitialCapacity: int(rt.containerConcurrency), // presume empty
```

Comment:
Why does this presume empty?

Reply:
empty as in `0 requests queued` rephrased it.